apps/editors/tests/test_views.py

```diff
@@ -2908,3 +2908,111 @@ def test_editors_abuse_report_page(self):
     def test_editors_review_page(self):
         url = reverse('editors.review', args=[self.addon.slug])
         self.assertNameAndNoXSS(url)
+
+
+class LimitedReviewerBase:
+    def create_limited_user(self):
+        limited_user = UserProfile.objects.create(username='limited',
+                                                  email="limited@mozilla.com")
+        limited_user.set_password('password')
+        limited_user.save()
+
+        permissions = [
+            {
+                'name': 'Add-on Reviewers',
+                'rules': 'Addons:Review',
+            },
+            {
+                'name': 'Limited Reviewers',
+                'rules': 'Addons:DelayedReviews',
+            },
+        ]
+        for perm in permissions:
+            group = Group.objects.create(name=perm['name'],
+                                         rules=perm['rules'])
+            GroupUser.objects.create(group=group, user=limited_user)
+
+    def login_as_limited_reviewer(self):
+        self.client.logout()
+        assert self.client.login(username='limited@mozilla.com',
+                                 password='password')
+
+
+class TestLimitedReviewerQueue(QueueTest, LimitedReviewerBase):
+
+    def setUp(self):
+        super(TestLimitedReviewerQueue, self).setUp()
+        self.expected_names = ['Nominated old']
+        self.url = reverse('editors.queue_nominated')
+
+        self.create_limited_user()
+        self.login_as_limited_reviewer()
+
+    def generate_files(self, subset=[]):
+        files = SortedDict([
+            ('Nominated new', {
+                'version_str': '0.1',
+                'addon_status': amo.STATUS_NOMINATED,
+                'file_status': amo.STATUS_UNREVIEWED,
+                'nomination': datetime.now()
+            }),
+            ('Nominated old', {
+                'version_str': '0.1',
+                'addon_status': amo.STATUS_NOMINATED,
+                'file_status': amo.STATUS_UNREVIEWED,
+                'nomination': datetime.now() - timedelta(days=1)
+            }),
+        ])
+        results = {}
+        for name, attrs in files.iteritems():
+            if not subset or name in subset:
+                results[name] = self.addon_file(name, **attrs)
+        return results
+
+    def test_results(self):
+        #import ipdb; ipdb.set_trace()
+        # `generate_files` happens within this test.
+        self._test_results()
```

Comment:
All those other tests are silently failing (I'll file a separate bug for that). I think you need to do this:
````
def test_results(self):
    self.expected_addons = self.generate_files().values()
    self._test_results()
````
and then fix the failures :)